core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeSorterSpillReader.java

```diff
@@ -17,21 +17,22 @@
 
 package org.apache.spark.util.collection.unsafe.sort;
 
+import java.io.*;
```

Comment:
let revert the unnecessary changes in this file

Reply:
Is it the `import order of the file header`, or from `org.slf4j.Logger & org.slf4j.LoggerFactory` to `org.apache.spark.internal.Logger &org.apache.spark.internal.LoggerFactory`
The later change is to prohibit the import of `org.slf4j.Logger & org.slf4j.LoggerFactory` in java code later, so as to prevent subsequent developers from introducing`org.slf4j.Logger & org.slf4j.LoggerFactory` again, thus writing inconsistent logic.

Reply:
Since there is no log migration in this file, why we need to change the import order here?

Reply:
The import order has been restored. 
I actually have an idea to align `the import order of java` with `the import order of scala`.
It's OK, I restored it, 😄